core/ir/riscv64/decode.c

```diff
@@ -121,33 +121,21 @@ decode_raw(dcontext_t *dcontext, byte *pc, instr_t *instr)
 bool
 decode_raw_is_jmp(dcontext_t *dcontext, byte *pc)
 {
-    /* FIXME i#3544: Not implemented */
-    ASSERT_NOT_IMPLEMENTED(false);
-    return false;
+    return ((*(uint *)pc & 0x7f) == 0x6f); /* JAL */
```

Comment:
Is there a #define or constant for JAL = 0x6f that can be used here?

Reply:
Currently, no. Should we #define one?

Reply:
Isn't it OP_jal, like the one in core/ir/riscv64/instr.c?

Reply:
well, OP_jal is an enum, it's used in the IR, and `x06f` here is a raw instruction pattern.